### DISCUSSION TITLE
Single variant tab not working!

### DISCUSSION THREAD

**Discussion by tkmamidi** [KarchinLab/open-cravat#260] [Usability & Feature Requests]
Single variant tab from the web UI is not working - [https://run.opencravat.org/webapps/variantreport/index.html](https://run.opencravat.org/webapps/variantreport/index.html)

**Comment by kmoad** [comment]
It requires a variant inputted in the top left. Variants can also be sent from the url. Try this example https://run.opencravat.org/webapps/variantreport/index.html?chrom=chr17&pos=41624191&ref_base=C&alt_base=T

I'm making some updates to make this more clear though, opening to a big blank page isn't very helpful.

**Comment by kmoad** [comment]
We've added documentation to the single variant tab when it first opens

![image](https://github.com/KarchinLab/open-cravat/assets/19417207/f44fec1a-efc0-4fe7-9af5-46f53c236ed6)
